tests/tier2/tc_2063_check_hypervisors_fqdn.py

```diff
@@ -0,0 +1,61 @@
+# coding:utf-8
+from virt_who import *
+from virt_who.base import Base
+from virt_who.register import Register
+from virt_who.testing import Testing
+
+
+class Testcase(Testing):
+    def test_run(self):
+        self.vw_case_info(os.path.basename(__file__), case_id='RHEL-188359')
+        self.vw_case_init()
```

Comment:
it's not necessary to run this case for libvirt-local and vdsm mode, so you should skip them as:
```
        hypervisor_type = self.get_config('hypervisor_type')
        if hypervisor_type in ('libvirt-local', 'vdsm'):
            self.vw_case_skip(hypervisor_type)
```

Reply:
ok